editor/components/block-list/block.js

```diff
@@ -108,8 +129,9 @@ class VisualEditorBlock extends Component {
 
 	componentDidUpdate( prevProps ) {
 		// Preserve scroll prosition when block rearranged
-		if ( this.previousOffset ) {
-			this.editorLayout.scrollTop = this.editorLayout.scrollTop +
+		const scrollContainer = getScrollContainer( this.node );
```

Comment:
I have a small concern about this getting called in each block update. Let's try and see if we notice perf issues.

Reply:
>I have a small concern about this getting called in each block update. Let's try and see if we notice perf issues.

This is very good to be concerned about, it shouldn't have been so generic, especially since `window.getComputedStyle` is a [non-trivial operation](https://gist.github.com/paulirish/5d52fb081b3570c81e3a). We can limit it to only the cases where a block has been moved by first checking `this.previousOffset`.